profile/README.md

```diff
@@ -1,12 +1,32 @@
-## Hi there 👋
 
-<!--
+#  Freedom Finance Stack
+![alt-text](Banner%20Image%20for%20FFS.png "FFS_Banner")
+
+## Overview
+
+Freedom Finance Stack is an open source initiative by RAZORPAY that aims to provide a
+comprehensive and interoperable solution for digital finance in India.
+It leverages the existing India Stack, a set of open-access software standards
+that facilitate digital payments, identity verification, dagit sta sharing, and consent management.
+Freedom Finance Stack builds on top of India Stack to offer additional features and capabilities.
+---
+
+## Mission
+* Freedom Finance Stack is an open source digital finance stack that aims to enable limitless innovation and collaboration in the financial ecosystem.
+* It offers solutions for different stakeholders, such as banks, technical service providers, institutions and regulators, and fintech.
+* It uses the world’s most powerful ACS (Access Control Server), which is EMVCo certified.
```

Comment:
It uses ? 
We don't use ACS, FFS provides ACS

Reply:
3rd line is different in context than other 3. It's not a mission. Create different section to show case current project we have under ffs and add ACS with one line description in that section